### ISSUE TITLE
Create Content Model for Bureau Data Sets

### ISSUE THREAD

**Issue by krisdelacruz** [18F/State-TalentMAP#599]
The content model will be a visual of a subset of the Bureau data:

1. Regional Bureau
2. Organization (It is not "Bureaus")
3. Other Bureaus (Offices)

**Comment by krisdelacruz** [comment]
@esgoodman I started on this by using the current data we have on the dev prototype. I will need to update this since its been updated. I wanted to have a running document of the recent changes via client feedback and suggestions so we are aware. Using Figma for this documentation. Please let me know if you are able to open it. https://www.figma.com/file/XxVv3hrqBJ8pXiXMJyoBbZgt/Search-Results-Content-Model

**Comment by esgoodman** [comment]
I can open it. @krisdelacruz based on our research, do we need to adjust? Or do the bureaus need a different set of filters than bidders?

**Comment by krisdelacruz** [comment]
@esgoodman Sorry for the confusion. I need to Retitle this. This is mainly a documentation of all the available filter data current on the prototype. We have been removing and adding filter data and I want to make sure we track those subtle changes. For example, we started with ~40+ tour of duties to 10 tours of duties (this is pre-usability testing).

**Comment by esgoodman** [comment]
@RyanSibley I'm assigning this to you because I think it needs a content strategist to evaluate the problem and break it down into doable tasks. I also think it's a useful entry point into the data we're working with on the project.  

@krisdelacruz expect @RyanSibley to ask you some questions about this before you go!

**Comment by RyanSibley** [comment]
@krisdelacruz hi there! I'm looking this over now. What a fascinating problem! I was looking through the history and see:

> For example, we started with ~40+ tour of duties to 10 tours of duties (this is pre-usability testing).

I'm curious, how did you whittle that down?

**Comment by krisdelacruz** [comment]
@ryanSibley From our first tackle on the data we got a spreadsheet from the client which was our first draft. This spreadsheet showcased the full extent of data (bidder side and admin side). We narrowed it down to 10 because these were the top tour of duties that users will see the most on the front-end. The other 30 were instants that the admin might be able to see.  =)

**Comment by RyanSibley** [comment]
Thank you so much. So the ten that remain are largely the only ones relevant to the bidder?

**Comment by krisdelacruz** [comment]
Correct, but we should preface that we need to test it. These were the top ten that we assumed will be best. The structure of the data is also something that has been continuously been used throughout the years (i.e. 2 YRS/TRANSFER, 2 YRS (1 R & R ) ) it is has been a 'staple' and we can recommend spelling it out it'll just be cumbersome on the UI it'll probably require two lines.

**Comment by RyanSibley** [comment]
Working on something that might help document why each data point is included: https://docs.google.com/document/d/1-Wz-NuQ2vJWoQXBzUOIb8XkGU2FM9yoOpuQ2TQsrv5Y/edit
